mixins/missing-options-mixin.html

```diff
@@ -72,13 +72,8 @@
           // assuming all objects from response array have same properties
           // we just need to make sure the first one is valid, no need to use a loop here
           let responseOptions = this.prepareReceivedOptions(response.detail);
-          this.set('options', [...options, ...responseOptions]);
+          this.set('options', [...this.options, ...responseOptions]);
 
-          // TODO: find a way to not use _updateAfterMissingOptionsReceived defined in other element,
-          // maybe a callback as param
-          if (responseOptions.length && typeof this._updateAfterMissingOptionsReceived === 'function') {
-            this._updateAfterMissingOptionsReceived(responseOptions);
-          }
```

Comment:
@adi130987 I removed this beucase when adding the missing values to the options list the `_selectedValuesOrOptionsChanged` observers triggers and sets `selectedItems` or `selectedItem` as the callbacks would.  It would actually cause a bug in `etools-dropdown-multi` where retrieved values would show twice

Reply:
:+1: